exonum-java-binding-time-service/src/main/java/com/exonum/binding/time/TimeSchema.java

```diff
@@ -17,28 +17,28 @@
 package com.exonum.binding.time;
 
 import com.exonum.binding.common.crypto.PublicKey;
-import com.exonum.binding.common.hash.HashCode;
-import com.exonum.binding.service.Schema;
 import com.exonum.binding.storage.indices.EntryIndexProxy;
 import com.exonum.binding.storage.indices.ProofMapIndexProxy;
 import java.time.ZonedDateTime;
-import java.util.List;
+import java.util.NoSuchElementException;
 
 /**
  * Exonum time service database schema.
  */
-public interface TimeSchema extends Schema {
+public interface TimeSchema {
 
   /**
-   * Returns stored time.
+   * Consolidated time output by the service, which can be used by other business logic on the
+   * blockchain.
+   * At the time when a new blockchain is launched, the consolidated time is unknown until the
+   * transactions from at least 2f + 1 validator nodes are processed.
+   *
+   * @throws NoSuchElementException if there is no consolidated time
```

Comment:
Is it true? This does not seem right to return ~ Optional *and* throw an exception if there is no value present.

I'd keep this method as is, not throwing exceptions, and consider (maybe, not in this PR) adding a method that _requires_ that the time is known and throws otherwise, like `getRequiredTime`.